pkg/config/template.go

```diff
@@ -937,6 +942,12 @@ const templateStringCrioRuntimeSeccompProfile = `# Path to the seccomp.json prof
 
 `
 
+const templateStringCrioRuntimePrivilegedSeccompProfile = `# Enable seccomp profiles for privileged containers.
+# This option supports live configuration reload.
+{{ $.Comment }}privileged_seccomp_profile = "{{ .PrivilegedSeccompProfile }}"
```

Comment:
we may want to mention that the value is considered as LocalhostRef.

...No one wants to set RuntimeDefault for privileged containers, do they?

Reply:
I don't think it's actually a required use case to use runtime default for privileged containers. But we could still add it later on if we want. Having it only a local path makes it much clearer from my point of view compared to parsing additional pseudo values like `RuntimeDefault` or such. 

I updated the docs to mention that it takes a file path.